src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -660,7 +660,7 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
             var requestMethodToClrMethodMap = new Dictionary<string, string>(StringComparer.Ordinal);
             var candidateAliases = new Dictionary<string, string>(StringComparer.Ordinal);
 
-            for (TypeInfo t = target.GetType().GetTypeInfo(); t != null; t = t.BaseType?.GetTypeInfo())
+            for (TypeInfo t = target.GetType().GetTypeInfo(); t != null && t != typeof(object).GetTypeInfo(); t = t.BaseType?.GetTypeInfo())
```

Comment:
What if target object overrides one of these methods? Should JsonRpc be allowed to call it?